### ISSUE TITLE
KtLint plugin failed to load reporter plain with Ktlint 0.49.0

### ISSUE THREAD

**Issue by DanySK** [JLLeitschuh/ktlint-gradle#665]
To reproduce:

set Ktlint version to 0.49.0 in the build file.

```
Exception is:
      org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':loadKtlintReporters'.
      	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.lambda$executeIfValid$1(ExecuteActionsTaskExecuter.java:149)
      ...
      Caused by: org.gradle.workers.internal.DefaultWorkerExecutor$WorkExecutionException: A failure occurred while executing org.jlleitschuh.gradle.ktlint.worker.LoadReportersWorkAction
      	at org.gradle.workers.internal.DefaultWorkerExecutor$WorkItemExecution.waitForCompletion(DefaultWorkerExecutor.java:283)
      ...
      Caused by: org.gradle.api.GradleException: KtLint plugin failed to load reporter plain.
      	at org.jlleitschuh.gradle.ktlint.worker.LoadReportersWorkAction.filterEnabledBuiltInProviders(LoadReportersWorkAction.kt:77)
      	at org.jlleitschuh.gradle.ktlint.worker.LoadReportersWorkAction.execute(LoadReportersWorkAction.kt:26)
      	at org.gradle.workers.internal.DefaultWorkerServer.execute(DefaultWorkerServer.java:63)
```

https://scans.gradle.com/s/5job5n3irzlie/console-log?task=:test#L187

**Comment by hanrw** [comment]
same issue.

**Comment by dsoneira** [comment]
Same problem with checkstyle reporter.

**Comment by caiodev** [comment]
Same problem with the html reporter

**Comment by wakingrufus** [comment]
I have reproduced the issue in my 0.49 support branch, and currently working on a solution using the new reporter api in 0.49

**Comment by knyttl** [comment]
Is there any progress on this?

**Comment by wakingrufus** [comment]
ktlint-gradle 11.4.0 is released which supports ktlint 0.49.1. please try this configuration

**Comment by DanySK** [comment]
Worked for me
